experimental/beacon_sim/beacon_sim.cc

```diff
@@ -554,6 +558,7 @@ int main(int argc, char **argv) {
       ("map_output_path", "Path to save map file to" , cxxopts::value<std::string>()->default_value(DEFAULT_MAP_SAVE_LOCATION))
       ("map_input_path", "Path to load map file from", cxxopts::value<std::string>()->default_value(DEFAULT_MAP_LOAD_LOCATION))
       ("load_off_diagonals", "Whether off diagonal terms should be loaded from map")
+      ("enable_brm_planner", "Enable planning after each step")
```

Comment:
May be worth matching the flag name with the description more explicitly. Flag name seems to indicate the use of the brm planner, flag description seems to indicate enabling planning in general